wandb/sdk/wandb_artifacts.py

```diff
@@ -728,22 +728,22 @@ def _ensure_can_add(self) -> None:
             raise ArtifactFinalizedError(artifact=self)
 
     def _add_local_file(
-        self, name: str, path: str, digest: Optional[B64MD5] = None
+        self, name: StrPath, path: StrPath, digest: Optional[B64MD5] = None
     ) -> ArtifactManifestEntry:
         with tempfile.NamedTemporaryFile(dir=get_staging_dir(), delete=False) as f:
             staging_path = f.name
             shutil.copyfile(path, staging_path)
             os.chmod(staging_path, 0o400)
 
         entry = ArtifactManifestEntry(
-            path=util.to_forward_slash_path(name),
+            path=name,
             digest=digest or md5_file_b64(staging_path),
             size=os.path.getsize(staging_path),
             local_path=staging_path,
         )
 
         self._manifest.add_entry(entry)
-        self._added_local_paths[path] = entry
+        self._added_local_paths[str(path)] = entry
```

Comment:
Nit: since `StrPath = str | os.PathLike[str]`, and a PathLike's `__str__` might pretty-print it or something,
```suggestion
        self._added_local_paths[os.fspath(path)] = entry
```

Reply:
Great suggestion!